image-index.md

```diff
@@ -1,12 +1,12 @@
-# OCI Image Manifest List Specification
+# OCI Image Index Specification
 
-The manifest list is a higher-level manifest which points to specific [image manifests](manifest.md) for one or more platforms.
-While the use of a manifest list is OPTIONAL for image providers, image consumers SHOULD be prepared to process them.
+The Image index is a higher-level manifest which points to specific [image manifests](manifest.md), ideal for one or more platforms.
```

Comment:
I'm not sure if we're punting copy-edits to a follow-up commit or not, but “Image” → “image” (either in this PR or in the follow-up copy-edit).